addon/components/session-publicationcheck.hbs

```diff
@@ -65,9 +65,9 @@
                 {{t "general.no"}}
               </td>
             {{/if}}
-            {{#if @session.objectives.length}}
+            {{#if (await @session.objectives)}}
```

Comment:
`has-many-length` here to make this sync.

Reply:
convered.